package.json

```diff
@@ -58,10 +58,10 @@
   "scripts": {
     "build": "./node_modules/babel-cli/bin/babel.js src --out-dir dist && echo '\n\\033[0;34m[+] \\033[0;32mBuild done\\033[0m'",
     "build:watch": "onchange 'src/**/*.js' -i -- yarn build",
+    "lint": "./node_modules/eslint/bin/eslint.js .eslint-bin src scripts",
```

Comment:
The only command that changed.
(The commands have been reordered in the alphabetical order)